src/utils/log2_fold_change.rs

```diff
@@ -0,0 +1,62 @@
+use std::ops::Not;
+
+use ordered_float::NotNan;
+
+use crate::utils::comparison::ComparisonOperator;
+use crate::variants::model::AlleleFreq;
+use std::fmt::{Display, Formatter};
+
+#[derive(Debug, Clone)]
+pub(crate) struct Log2FoldChange {
+    vaf_a: AlleleFreq,
+    vaf_b: AlleleFreq,
```

Comment:
Why does this hold two VAFs? I would have expected just the value, in a newtype like:

```rust
#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Derefable)]
pub(crate) struct Log2FoldChange(#[deref] NotNan<f64>);

impl Log2FoldChange {
    /// Calculate log2 fold change of a by b (i.e. (a / b).log2()).
    /// For a == b == 0.0, returns 0.0.
    /// Returns None if fold change cannot be calculated (if both a and b are zero).
    pub(crate) fn new(a: NotNan<f64>, b: NotNan<f64>) -> Self {
        if *a == 0.0 && *b == 0.0 {
            Log2FoldChange(NotNan::new(0.0).unwrap())
        } else {
            let value = a.log2() - b.log2();
            Log2FoldChange(NotNan::new(value).expect("bug: NaN when calculating Log2FoldChange"))
        }
    }
}
```

Sorry, I had forgotten to commit this code above. Also note the special handling of the zero case.

If the VAFs are for debugging, I think it would be better to handle that outside.